post-types/templates/page-profile-with-sidebar-and-table.php

```diff
@@ -81,18 +81,18 @@
 
             <?php
             if ($filter_map_id == '' && $metadata_dataset == '') { ?>
-                var get_od_selector_height = $('#od-selector').height();
```

Comment:
I have a couple of general comments on this section of code:

1) You've got basically the same thing in two different files. From a quick visual inspection, I can't tell if they're they are the same or if there are subtle but important differences between them. Keeping a ~100 line section of code in sync between two files  is not good practice.

2) In general, I think that there's a lot going on here that might be better done as a set of modern static CSS rules  and potentially a few small callbacks to set classes on elements when the behavior is supposed to change.

Reply:
@EricSoroos Thank you for your comment. I totally agree with you on the first comment. It is with script so I am not sure if it would working fine creating that as a template part, but I will give a try on that. Regarding the second comment, could you provide some more elaboration about your suggested approach?

By the way, the current code is trying to get the height of the elements via jQuery but it seems that it is not getting the exact height for the calculation.

Reply:
Well, something like https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky_positioning `position: sticky` would basically do everything that you're trying to do with this javascript in css. 

In general, you're going to be better off using the flow and the layout of the html to determine positioning rather than assembling heights of elements in javascript and trying to do it yourself.